src/main/java/com/github/black0nion/blackonionbot/commands/bot/AntiSpoilerCommand.java

```diff
@@ -1,47 +1,49 @@
 package com.github.black0nion.blackonionbot.commands.bot;
 
-import com.github.black0nion.blackonionbot.wrappers.jda.BlackGuild;
-import com.github.black0nion.blackonionbot.wrappers.jda.BlackMember;
-import com.github.black0nion.blackonionbot.wrappers.jda.BlackUser;
-import com.github.black0nion.blackonionbot.commands.TextCommand;
-import com.github.black0nion.blackonionbot.commands.CommandEvent;
+import com.github.black0nion.blackonionbot.commands.SlashCommand;
+import com.github.black0nion.blackonionbot.commands.SlashCommandEvent;
 import com.github.black0nion.blackonionbot.systems.antispoiler.AntiSpoilerType;
-import com.github.black0nion.blackonionbot.systems.language.LanguageSystem;
 import com.github.black0nion.blackonionbot.utils.EmbedUtils;
 import com.github.black0nion.blackonionbot.utils.Placeholder;
-import com.github.black0nion.blackonionbot.utils.Utils;
+import com.github.black0nion.blackonionbot.wrappers.jda.BlackGuild;
+import com.github.black0nion.blackonionbot.wrappers.jda.BlackMember;
+import com.github.black0nion.blackonionbot.wrappers.jda.BlackUser;
 import net.dv8tion.jda.api.Permission;
-import net.dv8tion.jda.api.entities.Message;
 import net.dv8tion.jda.api.entities.TextChannel;
-import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.Commands;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import org.jetbrains.annotations.NotNull;
 
-public class AntiSpoilerCommand extends TextCommand {
+public class AntiSpoilerCommand extends SlashCommand {
+	private static final String STATUS = "status";
 
 	public AntiSpoilerCommand() {
-		this.setCommand("antispoiler", "as")
-			.setSyntax("[replace | delete | off]")
-			.setRequiredPermissions(Permission.MESSAGE_MANAGE);
+		super(builder(Commands.slash("antispoiler", "Used to deleted/disable/replace the anti-spoiler system.")
+				.addOptions(new OptionData(OptionType.STRING, STATUS, "The status", true)
+						.addChoice("replace", "replace")
+						.addChoice("delete", "delete")
+						.addChoice("off", "off")))
```

Comment:
Please make this optional to allow checking the status of the system (see below)